### PR TITLE
integration_test (ticdc): set global `tidb_enable_clustered_index = INT_ONLY` before execute any SQL

### LINKED ISSUE #7535: some integration test cases fail
### Which jobs are flaking?

-

### Which test(s) are flaking?

-

### Jenkins logs or GitHub Actions link

img_v2_f6827c78-0dff-4aa4-a20d-da621c80ab6g

### Anything else we need to know

- Does this test exist for other branches as well?

- Has there been a high frequency of failure lately?

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #7535

### What is changed and how it works?

After https://github.com/pingcap/tidb/pull/37384 was merged, `tidb_enable_clustered_index` is `ON` by default. 
Some TiCDC integration test cases need to drop a table's primary key, but if the primary key is a clustered index, it can't be droped. 
This PR set `global tidb_enable_clustered_index = INT_ONLY` when we connect to upstream TiDB in integration test, which is a workaround to make the integration test cases works as before.

#### Tests 

 - Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: tests/integration_tests/_utils/run_sql [hunk 1]
```diff
@@ -25,5 +25,9 @@ if [ $# -gt 1 ]; then
 	other=${*}
 fi
 
+prepare="set global tidb_enable_clustered_index = 'int_only';"
```
Comment (hi-rustin) [inline tests/integration_tests/_utils/run_sql:28]: Could you please add a comment for it?

**Review by hi-rustin — APPROVED** [review]
Thanks!

**Comment by hi-rustin** [comment]
Please refine your PR title and body. Thanks!

**Review by amyangfei — APPROVED** [review]

**Pull request merged**